scene/animation/animation_tree_player.cpp

```diff
@@ -815,9 +814,9 @@ void AnimationTreePlayer::_process_animation(float p_delta) {
 					case Animation::TYPE_VALUE: { ///< Set a value in a property, can be interpolated.
 
 						if (a->value_track_is_continuous(tr.local_track)) {
-							Variant blended, value = a->value_track_interpolate(tr.local_track,anim_list->time);
-							Variant::blend(tr.track->object->get(tr.track->property),value,blend,blended);
-							tr.track->object->set(tr.track->property,blended);
+							Variant value = a->value_track_interpolate(tr.local_track,anim_list->time);
```

Comment:
Aren't you creating a local variable ``value`` here, thus the one you added in the class definition stays initialized? Or does the initialization happen within ``Variant::blend()`` on l. 818?